infra/terraform/base_stacks/README.md

```diff
@@ -0,0 +1,34 @@
+https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3157458948/Base+CloudFormation+Stacks
+
+
+Build Account Only:
+- GitHub Identity
+- Signer (also dev to allow deployments there?)
+- Container Signer
+
+Per Account:
+- API Gateway logs
+- Certificate expiry
+- Grafana?
+- Checkov hook
+- lambda audit hook
+- infra audit hook
+
+- identity broke oidc provider ( different from github identity?)
+
+- ecr scan result logger?
```

Comment:
this stack isn't great - it sends a very complex email that I find hard to read. I suggest we remove it if it has been deployed